### ISSUE TITLE
[FEATURE] Package for Arch

### ISSUE THREAD

**Issue by alexVinarskis** [alexVinarskis/dell-powermanager#25]
**What is suggested Feature, and what problem it solves?**
Package for Arch, to .aur

Debian dependencies were already conditioned to debian based systems only, app can run from source on Arch and other distros.

**Background info**
OS - Arch Linux

**Comment by meduk0** [comment]
to make things ez you can set a distrobox with ubunto (it worked with 23.04 for me ) i like the app and i think about contributing as i need some of the functions it gives .
just follow this :
# i use podman you can use docker if you love it (i don't) 
``` sudo pacman -S podman distrobox  ```
# enter distrobox 
```distrobox enter ubuntu-23-04 ```
 #then all of the following is in ubuntu (inside arch )
```sudo apt install git
git clone https://github.com/alexVinarskis/dell-powermanager
cd dell-powermanager/
/package.sh ```
and then just launch it  :
``` dell-powermanager ```
it is very simple and it doesn't consume alot of power (barely 1.5w according to powertop ) and work on evry single linux distro as long you can get distrobox git and podman (or any container ) 
@alexVinarskis you might need to read this as it provide your software without any tweaking or knowledge of other distros 
i think i will get it on the aur anyways (i am busy right now but i will do it as close as i can )

**Comment by alexVinarskis** [comment]
Hey,

Thanks for the input. Indeed its a bit of a left out feature...
I am familiar with dockerized app approach, although havent used distrobox specifically.

Actually, I am planning on distributing app via `flatpak`, which is basically (in simple terms) same idea of containerizing apps but in more user friendly manner,and it is also widely adopted by multiple distros. Which is why I will not be making native arch/fedora packages.

Moreover, the process of packaging (in this case to .deb, which limits it to ubuntu or debian) only moves binaries to expected directories and adds user friendly things like desktop shortcut. You can see in detail the [packing ](https://github.com/alexVinarskis/dell-powermanager/blob/master/package.sh)script.

I also configured CI to release just bin bundle (eg. [here](https://github.com/alexVinarskis/dell-powermanager/releases/tag/0.10.0), the `.tar.xz` file). This means that you do not in fact even need containerization, its a bit of an overkill haha. Simple untar the archive, and run `dell_powermanager`executable on practically any linux distro :)

**Comment by meduk0** [comment]
yeah true , i work alot with containers these days so i said why not (just wanna save my self some pain as porting a software from debian to arch is very hard (arch push updates every single day (at least for some pkgs ) while debian is in a stable side) 
i am with the idea of flatpak as it is a software that gonna be used once or twice a day so no reason for juicing performance out of it )
you have done a pretty good job it helps a ton (i used on a dell g15 5530 ) . I never worked with hardware specific in years and i am willing to help in testing or maintaining things .

**Comment by alexVinarskis** [comment]
Glad to hear it works as intended!

If you could submit a compatibility report for your device (there is a template when submitting a bug) i will add it list to successfully tested devices

**Comment by becksy82** [comment]
I would love it, if it will add to the aur.
